src/views/mfa-verify/PassCodeForm.js

```diff
@@ -68,12 +76,22 @@ define(['okta', 'q', 'views/shared/TextBox'], function (Okta, Q, TextBox) {
       return this.model.appState.get('isMfaChallenge') && this.model.get('answer');
     },
 
+    showWarning: function (msg) {
+      clearTimeout(warningTimeout);
```

Comment:
no need to `clearTimeout` here. Actually I don't think you need the variable `warningTimeout` anymore